docs/retrieve.txt

```diff
@@ -0,0 +1,360 @@
+
+         </body>
+         </html>
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+
+      return view('browse_movies', [
+         'movies' => DB::collection('movies')
+               ->where('imdb.votes', '>', 1300000)
+               ->get()
+      ]);
+
+   .. output::
+      :language: none
+      :visible: false
+
+      Title: The Shawshank Redemption
+      Year: 1994
+      Runtime: 142
+      IMDB Rating: 9.3
+      IMDB Votes: 1513145
+      Plot: Two imprisoned men bond over a number of years, finding solace and eventual redemption
+      through acts of common decency.
+
+      Title: The Dark Knight
+      Year: 2008
+      Runtime: 152
+      IMDB Rating: 9
+      IMDB Votes: 1495351
+      Plot: When the menace known as the Joker wreaks havoc and chaos on the people of Gotham, the
+      caped crusader must come to terms with one of the greatest psychological tests of his ability to
+      fight injustice.
+
+.. _laravel-retrieve-all:
+
+Retrieve All Documents in a Collection
+--------------------------------------
+
+You can retrieve documents without specifying a query filter, which returns all
+documents in a collection.
+
+To retrieve all documents, call the ``get()`` method on an Eloquent model that represents
+your collection. Alternatively, you can use the ``get()`` method's alias ``all()`` to perform
+the same operation.
+
+.. _laravel-retrieve-all-example:
+
+Retrieve All Documents
+~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following syntax to run a find operation that matches all documents:
+
+.. code-block:: php
+
+   $movies = Movie::get();
+
+.. warning::
+
+   The ``movies`` collection in the Atlas sample dataset contains a large amount of data.
+   Retrieving and displaying all documents in this collection might cause your web 
+   application to time out.
+
+.. _laravel-modify-find:
+
+Modify Behavior
+---------------
+
+You can modify the results of a find operation by chaining the following methods
+to ``where()``:
+
+- ``skip()``: sets the number of documents to skip when returning results
+- ``take()``: sets the total number of documents to return
+- ``first()``: returns the first document that matches the query filter
+
+Skip and Limit Results
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following example queries for documents in which the ``year`` value is ``1999``.
+The operation skips the first ``2`` matching documents and outputs a total of ``3``
+documents.
+
+.. important:: Required File Changes
+
+   To run the code example, you must edit the ``MovieController.php`` file in your
+   Laravel Quick Start application. Copy the code example and paste it into the ``show()``
+   function, replacing any existing code inside this function.
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: php
+
+      return view('browse_movies', [
+         'movies' => Movie::where('year', 1999)
+            ->skip(2)
+            ->take(3)
+            ->get()
+      ]);
```

Comment:
Same separation of concerns. I'm not sure the `return view` makes much sense here.
```suggestion
      $movies = Movie::where('year', 1999)
          ->skip(2)
          ->take(3)
          ->get()

      return view('browse_movies', [
         'movies' => $movies,
      ]);
```